File: src/CorporateMessengerWithAgile/Application/Dto/SprintDto.cs
namespace Application.Dto
{
    public class SprintDto : BaseDto
    {
        public Guid TeamId { get; set; }
        public DateTime DateStart { get; set; }
        public DateTime DateEnd { get; set; }
        public IReadOnlyList<Guid> TaskItemIds { get; set; } = null!;
    }
}

File: src/CorporateMessengerWithAgile/ReactApp/reactapp.client/src/models/entity/SprintDto.ts
// src/models/entity/SprintDto.ts
import { z } from 'zod';
import { BaseDtoSchema } from './BaseDto';
import { GuidSchema } from '../Guid';

export const SprintDtoSchema = BaseDtoSchema.extend({
    teamId: GuidSchema,
    dateStart: z.coerce.date(),
    dateEnd: z.coerce.date(),
    taskItemIds: z.array(GuidSchema).optional(),
});

export type SprintDto = z.infer<typeof SprintDtoSchema>;
